app/registries/foreman/plugin/graphql_plugin_fields.rb

```diff
@@ -6,15 +6,21 @@ module GraphqlPluginFields
       module ClassMethods
         def realize_plugin_query_extensions(source = Foreman::Plugin.graphql_types_registry.plugin_query_fields)
           source.map do |plugin_type|
-            send plugin_type[:field_type], plugin_type[:field_name], plugin_type[:type]
+            send plugin_type[:field_type], plugin_type[:field_name], constantize_type(plugin_type[:type])
           end
         end
 
         def realize_plugin_mutation_extensions(source = Foreman::Plugin.graphql_types_registry.plugin_mutation_fields)
           source.map do |plugin_type|
-            send :field, plugin_type[:field_name], mutation: plugin_type[:mutation]
+            send :field, plugin_type[:field_name], mutation: constantize_type(plugin_type[:mutation])
           end
         end
+
+        def constantize_type(string)
```

Comment:
you can use this instead
https://github.com/theforeman/foreman/blob/develop/app/services/foreman/module.rb